src/SourceControl-GitHub/APIWrapper.ts

```diff
@@ -0,0 +1,101 @@
+import { Octokit } from "octokit";
+import { Issue } from "./Issue";
+import { Milestone } from "./Milestone";
+import { Organization } from "./Organization";
+import { PullRequest } from "./PullRequest";
+import { Repository } from "./Repository";
+
+// authenticate via personal token for now
+const octokit = new Octokit({ auth: process.env._AUTHTOKEN });
```

Comment:
should we have a throw errors if required env variables are not available?

Reply:
yeah, silent failures suck